### PR TITLE
Optional draw

### PR DESCRIPTION
**Pull request by jwildfire**
Quick PR to address #125. 

Not sure if we want to do this in v1.11, or wait and do a quick v1.11.1 once this is done. Fairly low risk, but probably worth having @danedexF5 do a general QC of the controls for this one since it is changing some core functionality.

### CODE REVIEW

File: src/controls/checkRequired.js [hunk 1]
```diff
@@ -13,5 +13,8 @@ export default function checkRequired(dataset) {
                     '" does not match any column in the provided dataset.'
             );
         }
+
+        //draw the chart when a control changes unless the user specifies otherwise
+        d.draw = e.draw == undefined ? true : e.draw;
```
Comment (samussiah) [inline src/controls/checkRequired.js]: This should be `e.draw` not `d.draw` right?

File: src/controls/makeSubsetterControl.js [hunk 2]
```diff
@@ -73,15 +73,16 @@ export default function makeSubsetterControl(control, control_wrap) {
...
+                draw: true
             };
             targets.forEach(e => {
                 setSubsetter(e, new_filter);
                 //call callback function if provided
                 if (control.callback) {
                     control.callback();
                 }
-                e.draw();
+                if (control.draw) e.draw();
```
Comment (samussiah) [inline src/controls/makeSubsetterControl.js]: So the use case is only for subsetter controls?

File: src/controls/checkRequired.js (hunk 1 above)
Comment (jwildfire) [reply src/controls/checkRequired.js]: yep - good catch.

File: src/controls/makeSubsetterControl.js (hunk 2 above)
Comment (jwildfire) [reply src/controls/makeSubsetterControl.js]: No, it was for all controls - might've missed a spot though. I thought the update in `scr/controls/changeOption.js` covered the other use case - no?

Comment (samussiah) [reply src/controls/makeSubsetterControl.js]: quite right, quite right

**Pull request merged**